token-upgrade/cli/src/main.rs

```diff
@@ -336,7 +334,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
                     .value_parser(SignerSourceParserBuilder::default().allow_all().build())
                     .value_name("MULTISIG_SIGNER")
                     .takes_value(true)
-                    .multiple(true)
+                    .action(ArgAction::Append)
```

Comment:
The documentation seems to imply that we should use `multiple_occurrences` instead, ie:

> Arg::multiple_occurrences in favor of Arg::multiple_values (positional), ArgAction::Append (flag with value), or ArgAction::Count (flag without value)

from https://github.com/clap-rs/clap/pull/3797, and:

> Replace Arg::multiple(bool) with Arg::multiple_values / Arg::multiple_occurrences

from https://github.com/clap-rs/clap/releases/tag/v3.0.0

Is there a reason to use your approach instead?

Reply:
Yeah `multiple` was deprecated in favor of `multiple_occurrences` in `v3.0.0`, but `multiple_occurrences` was deprecated in favor of `ArgAction` in `v3.2.0` (https://github.com/clap-rs/clap/releases/tag/v3.2.0) 😅.

Reply:
Haha I had a feeling, thanks for the info!